tests/testTrello.spec.js

```diff
@@ -0,0 +1,97 @@
+import { test, expect } from '@playwright/test';
+const  credentials = require('../cypress/fixtures/credentials1.json');
+const { chromium } = require('@playwright/test');
+import { Commands } from './pages/commandsUi.page';
+import { ApiCommands } from './pages/apiCommands.page';
+import { LoginPage } from './pages/loginPage.page';
+const axios = require('axios');
+
+
+test.describe('Test suite', async () => {
+
+    let browser, context, page;
+
+    test.beforeAll(async () => {
+        browser = await chromium.launch();
+        context = await browser.newContext();
+        page = await context.newPage();
+        const userMethod = new LoginPage(page);
+
+        await userMethod.gotoPage();
+        await userMethod.enterEmail(credentials.email);
+        await userMethod.enterPassword(credentials.pw);
+        await expect(page).toHaveURL(`u/${credentials.userName}/boards`);
+    });
+
+
+    test.afterAll(async () => {
+        await browser.close();
+    });
+
+    test.describe('create a new board', async () => {
+        test('create a board whit UI', async () => {
+        const userMethod = new Commands(page);
```

Comment:
Why the name of this variable is userMethod?
Just use readable and descriptive variable names
```
const workspacePage = new WorkspacePage(page);
```